lmdeploy/vl/model/phi3_vision.py

```diff
@@ -119,11 +118,20 @@ def _process_image_embedding(self, pixel_values: torch.Tensor,
 
 
 @VISION_MODELS.register_module()
-class Phi3VisionModel(VisonModel):
+class Phi3VisionModel(LlavaHfVisionModel):
     """Llava hf vision model."""
 
     _arch = 'Phi3VForCausalLM'
 
+    def build_preprocessor(self):
+        processor = AutoProcessor.from_pretrained(self.model_path,
+                                                  trust_remote_code=True)
+        if hasattr(processor, 'tokenizer'):
+            del processor.tokenizer
+            processor.prtokenizer = None
```

Comment:
`prtokenizer` -> `tokenizer`

Reply:
In latest main, it is `prtokenizer`
why `del` and set `None`? I think either one is fine, isn't it?

Reply:
it's a typo. Since `tokenizer` for text is not used and we only use the image_processsor, so it's deleted.